holmes/config.py

```diff
@@ -612,9 +613,15 @@ def merge_and_override_bultin_toolsets_with_toolsets_config(
                 toolsets_with_updated_statuses[toolset.name].override_with(toolset)
             else:
                 try:
-                    validated_toolset = YAMLToolset(
-                        **toolset.model_dump(exclude_none=True)
-                    )
+                    tags = toolset.tags
+                    if ToolsetTag.MCP in tags:
+                        validated_toolset = RemoteMCPToolset(
```

Comment:
Can we try to check if the field url is included and then try to parse it as remote mcp tool instead? Usage of tags here feels like we are twisting the meaning of them.

If any objections let's discuss it.